diff --git a/setup/src/Magento/Setup/Console/CommandList.php b/setup/src/Magento/Setup/Console/CommandList.php
index ae65e82..26770ab 100644
--- a/setup/src/Magento/Setup/Console/CommandList.php
+++ b/setup/src/Magento/Setup/Console/CommandList.php
@@ -69,7 +69,8 @@ class CommandList
             \Magento\Setup\Console\Command\RollbackCommand::class,
             \Magento\Setup\Console\Command\UpgradeCommand::class,
             \Magento\Setup\Console\Command\UninstallCommand::class,
-            \Magento\Setup\Console\Command\DeployStaticContentCommand::class
+            \Magento\Setup\Console\Command\DeployStaticContentCommand::class,
+            \ImaginationMedia\XmlValidator\Console\Command\ValidateXmlCommand::class
         ];
     }
